dpe/fuzz/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+# Licensed under the Apache-2.0 license
+
+[package]
+name = "dpe-fuzz"
+version = "0.0.0"
+publish = false
+edition = "2021"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+libfuzzer-sys = "*"
+log = "*"
+simplelog = "*"
```

Comment:
Same, let's version these

Reply:
Marking as done. Crates are now merged into one, so this is now a duplicate.